packages/utilities/psammead-styles/src/typography.jsx

```diff
@@ -0,0 +1,17 @@
+import { shape, func } from 'prop-types';
```

Comment:
Could we move this file to `typography.stories.jsx` since it is a helper component.

Reply:
My thinking behind having the TypographyText component in a different file was to separate concerns. Personally, I think they should stay separate

Reply:
I saw that we already have test covering typography in https://github.com/bbc/psammead/blob/e42de7c76b31851de995a6823c34c7b0be90f636/packages/utilities/gel-foundations/tests/getTypeSizes.test.js#L5-L25

By having `typography.test.jsx` we would be duplicating the above test that ensures typography function works as expected.
This means we won't need `typography.test.jsx`. If we delete that file, now `typography.jsx` will only be needed in `typography.stories.jsx`. Thats why i made the recommendation.

Reply:
👌Changed it.